t/apicast-policy-upstream_mtls.t

```diff
@@ -575,3 +575,72 @@ ssl_client_i_dn: CN=localhost,OU=APIcast,O=3scale
 --- error_code: 200
 --- no_error_log
 [error]
+
+
+=== TEST 8: MTLS policy with correct CA certificate, but invalid host
```

Comment:
Where are you specifying the invalid host here?

Reply:
Ok, can we add a comment to the test adding this information, ie. the certificates are set up for `localhost` but here we will use a different hostname?